test/dlc_tests/container_tests/bin/pytorch_tests/testPyTorchNccl

```diff
@@ -13,7 +13,7 @@ export OMP_NUM_THREADS=${NUM_GPU}
 
 TRAINING_LOG=${LOG_DIR}/pytorch_train_nccl.log
 
-echo "Training resnet18 using PyTorch Nccl... This may take a few minutes. You can follow progress on the log file : $TRAINING_LOG"
+echo "Training resnet18 using PyTorch Nccl.., This may take a few minutes. You can follow progress on the log file : $TRAINING_LOG"
```

Comment:
nit: no need for this line code change

Reply:
Added this to allow testing of the new image to happen. Will revert it.

Reply:
Make sense. Thanks